site/gatsby-site/src/utils/AlgoliaUpdater.js

```diff
@@ -59,9 +63,12 @@ class AlgoliaUpdater {
       for (const report_number of incident.reports) {
         const report = reports.find((r) => r.report_number == report_number);
 
+        const text = await remark().use(remarkStrip).process(report.text);
```

Comment:
Should use the plain_text field instead, no?

Reply:
Checked again and no, because that `report` object is built from the `reports` collection + the translated `text`. This will need to be updated later to take into account the `report.language` field though.